Plugin/src/SofaPython3/DataHelper.cpp

```diff
@@ -605,6 +617,7 @@ BaseLink* addLink(py::object py_self, const std::string& name, py::object value,
 
     BaseLink::InitLink<Base> initlink(self, name, help);
 
+    // TODO: replace new keyword with a reference counted pointer (make_unique or make_shared)
```

Comment:
why not doing it instead of adding a TODO ?

Reply:
@damienmarchal Because it does not work with the current version of pybind.
See the comments above with @alxbilger 

It would be great to just upgrade everything to python3.9 and pybind11 2.6.1